src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -874,13 +874,16 @@ NetRouteMap FastRouteCore::run()
     logger_->info(GRT, 101, "Running extra iterations to remove overflow.");
   }
 
-  static const int max_overflow_increases_ = 15;
+  static const int max_overflow_increases_ = 25;
```

Comment:
Remove the trailing _ as this isn't a class member.  There is also no need for it to be static since it is a const.